app/services/XmlBuilderService.scala

```diff
@@ -204,21 +214,31 @@ object XmlBuilderService {
   }
 
   private def addChildrenToRoot(root: Node, childNodes: NodeSeq): Node =
-    Elem(
-      root.prefix,
-      root.label,
-      root.attributes,
-      root.scope,
-      root.child.isEmpty,
-      root.child ++ childNodes: _*
-    )
+    Elem(root.prefix, root.label, root.attributes, root.scope, root.child.isEmpty, root.child ++ childNodes: _*)
 
   private def buildOptionalElem[A](value: Option[A], elementTag: String): NodeSeq = value match {
-    case Some(result: String)    => loadString(s"<$elementTag>$result</$elementTag>")
+    case Some(result: String) => {
+      val encodeResult = StringEscapeUtils.escapeXml11(result)
+      loadString(s"<$elementTag>$encodeResult</$elementTag>")
+    }
+    case _ => NodeSeq.Empty
+  }
+
+  private def buildOptionalDate(value: Option[LocalDate], elementTag: String): NodeSeq = value match {
     case Some(result: LocalDate) => loadString(s"<$elementTag>${Format.dateFormatted(result)}</$elementTag>")
     case _                       => NodeSeq.Empty
   }
 
+  private def buildAndEncodeElem[A](value: A, elementTag: String): NodeSeq = value match {
```

Comment:
I like how you're using the type to determine if encoding is needed. Do you think we should create a new type for _Language Code_ We know it's only ever EN and if we have a custom type we can also exclude it from any encoding. Thoughts?